src/App.vue

```diff
@@ -210,6 +213,35 @@ export default {
         console.warning("fail to open service");
       }
     },
+    filterTag: function (filter) {
+      this.showMenu = true;
+      this.$nextTick(() => {
+        document.getElementById("searchBox").focus();
```

Comment:
May [`ref`](https://vuejs.org/v2/guide/components-edge-cases.html#Accessing-Child-Component-Instances-amp-Child-Elements) be used to avoid using `nextTick` and `getElementById`? (and get a simplier, clean code)

Reply:
If you can get it to work! I tried using `ref` first and I was having issues for some reason.

Reply:
no worries then, let's keep it this way :+1: